packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids.ts

```diff
@@ -137,13 +137,36 @@ export const connectedAccountStandardFieldIds = {
   calendarChannels: '20202020-af4a-47bb-99ec-51911c1d3977',
 };
 
-export const eventStandardFieldIds = {
+export const auditLogStandardFieldIds = {
+  name: '20202020-2462-4b9d-b5d9-745febb3b095',
+  properties: '20202020-5d36-470e-8fad-d56ea3ab2fd0',
+  context: '20202020-b9d1-4058-9a75-7469cab5ca8c',
+  objectName: '20202020-76ba-4c47-b7e5-96034005d00a',
+  objectId: '20202020-c578-4acf-bf94-eb53b035cea2',
+  workspaceMember: '20202020-6e96-4300-b3f5-67a707147385',
+};
+
+export const behavioralEventStandardFieldIds = {
```

Comment:
what is behavioralEventStandardFieldIds?

Reply:
Behavioral Events is an object for Segment-like events (e.g. coming from website)